lib/core/src/system/web/dom/html/css3d_system.rs

```diff
@@ -0,0 +1,39 @@
+use crate::prelude::*;
+
+use super::Css3dObject;
+use crate::display::object::DisplayObjectData;
+use crate::display::world::World;
+use crate::system::web::Result;
+use crate::system::web::dom::html::Css3dRenderer;
+
+use logger::Logger;
+
+/// Css3dSystem enables us to create instances of HtmlElement objects in the 3d world.
+#[derive(Debug)]
+pub struct Css3dSystem {
+    display_object : DisplayObjectData,
+    css3d_renderer : Css3dRenderer,
+    logger         : Logger
+}
+
+impl Css3dSystem {
+    /// Creates a new instance of Css3dSystem.
+    pub fn new(world:&World) -> Self {
+        let scene          = world.scene();
+        let css3d_renderer = scene.css3d_renderer();
+        let logger         = css3d_renderer.logger().sub("Css3dSystem");
+        let display_object = DisplayObjectData::new(&logger);
```

Comment:
You can find this pattern in several files of mine. For example in Mesh, where I create loggers for its children. The reason why I do it is that the Mesh responsibility is to describe the responsibility of its children. Like, it creates 4 scopes telling to them - you are point scope, you are vertex scope etc. Here we've got a completely different story. Parent does not tell its child about the child responsibility, so parent sohuld bnot create the logger for its child. Child should do it instead in its constructor.

Reply:
Please check the refactoring above where I move the construction of `Css3dSystem` to `Css3dRenderer`. Does it still apply?